src/kinova_gen3_mujoco_config/config/config.yaml

```diff
@@ -72,6 +72,7 @@ hardware:
       - external_camera: "true"
       - wrist_realsense: "true"
       - use_internal_bus_gripper_comm: "false"
+      - mujoco_model: "description/mujoco/landsat_scene.xml"
```

Comment:
Thanks for adding this. One small change: I feel like the `scene.xml` model should be the default.